### PR TITLE
Update fr.yml

### PR DESCRIPTION
**Pull request by TheIntelloBox**

### CODE REVIEW

File: src/main/resources/localizations/fr.yml [hunk 1]
```diff
@@ -4,61 +4,61 @@ mcb:
...
+          all-invalid-flags: "Tous les flags indiqués sont invalides."
+          success: "Permission(s) mise(s) avec succès pour le role %s"
+          omitted-flags: "Le(s) flag(s) suivant(s) est/sont invalide(s) et ont été ommis: %s"
+          added: "Les flags suivants ont été ajoutés: %s"
         check:
-          description: "Check the current permission flags of a role."
-          all-description: "All roles that have permission flags and their corresponding flags"
+          description: "Regarder les permissions actuelles pour un rôle."
+          all-description: "Tous les rôles qui ont des permissions et leurs flags."
           success: "%s permission flags."
```
Comment (TheFlagCourier) [inline src/main/resources/localizations/fr.yml]: Untranslated? May not need to be though...

**Review by TheFlagCourier — CHANGES_REQUESTED** [review]
Line 25 seems to be untranslated.

File: src/main/resources/localizations/fr.yml (hunk 1 above)
Comment (CyR1en) [reply src/main/resources/localizations/fr.yml]: %s is the role in this case. So maybe translate it to `indicateur d'autorisation du %s`. I'm not good with french :3.

**Comment by TheIntelloBox** [comment]
I did it now.

**Review by TheFlagCourier — APPROVED** [review]

**Comment by TheFlagCourier** [comment]
Alright, merging into the Lang-Update branch for later merge.

**Pull request merged**